book.rb

```diff
@@ -0,0 +1,16 @@
+require './item'
+
+class Book < Item
+  attr_reader :publisher, :cover_state
+
+  def initialize(publisher, cover_state, publish_date)
+    super(publish_date)
+    @publisher = publisher
+    @cover_state = cover_state
+    @archived = false
+  end
+
+  def can_be_archived?
+    super || cover_state == 'bad'
+  end
```

Comment:
- [ ] According to the UML diagram, the `can_be_archived?` method should be a **private method**. What do you think about making it one?

Reply:
You are right @rpire thank you for mentioning it. Solved as you can see here:
![image](https://user-images.githubusercontent.com/60292761/143017776-99ca730d-a02d-4927-8a27-efe2492ff2a9.png)

Reply:
Done 👍